common/changes/office-ui-fabric-react/remove-input-bottom-margin_2017-05-31-16-01.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Inputs: removed bottom margin causing issues when putting more than one per line",
+      "type": "minor"
```

Comment:
While this is technically a breaking change, I can see the hesitancy in making a major version bump because of it. I wonder where we draw that line between a breaking change and a **BREAKING CHANGE**.